presto-docs/src/main/sphinx/release/release-329.rst

```diff
@@ -0,0 +1,58 @@
+=========================
+Release 329 (XXX 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix incorrect result of ``last_day_of_month`` function for first day of month. (:issue:`2452`)
+* Handle common disk failures during spill. When one of disks got broken, it
+  Presto will use another one without failing the query. (:issue:`2444`)
+* Improve performance and reduce load on external systems when
+  querying ``information_schema``. (:issue:`2488`)
+* Limit broadcasted table size to 100MB by default when using the ``AUTOMATIC``
+  join type selection strategy. (:issue:`2527`)
+* Hide non-standard columns ``comment`` and ``extra_info`` in the standard
+  ``information_schema.columns`` table. These columns can still be selected,
+  but will no longer appear when describing the table. (:issue:`2306`)
+* Enable ``AUTOMATIC`` join reordering and join type selection strategy by default. (:issue:`2528`)
```

Comment:
I think it would be more informative if supplemented with "CBO" and "Cost-Based Optimizer".
Otherwise this is very easy to overlook

Reply:
I don't like using the term CBO -- it's too opaque. And "cost-based optimizer" is a little too generic, since it's not a general purpose cost-based optimizer. It only applies to join reordering and type selection. Maybe we can just link to the corresponding sections under https://prestosql.io/docs/current/optimizer/cost-based-optimizations.html